content/cloud/custom-domains-tutorial.md

```diff
@@ -0,0 +1,71 @@
+
+## Prerequisites
+
+To ensure the tutorial goes smoothly, please ensure you have the following: 
+
+* Most importantly, you must have a domain you own. If you don't have a domain, please visit a domain registrar like [Namecheap](https://www.namecheap.com/) or [Google Domains](https://domains.google/?gad=1&gclid=Cj0KCQjwzdOlBhCNARIsAPMwjbyNg3a3GcrQjbs3vBSDg4AWnnEkc96mJupyg8nqqiKyxVCyaePRXfAaAjRLEALw_wcB&gclsrc=aw.ds) to purchase one. 
+* A Fermyon Cloud account that is set up via your preferred [GitHub user account](https://docs.github.com/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/remembering-your-github-username-or-email)
+* A Spin application you would like to apply the custom domain to. If you aren't familiar with how to deploy an application on Fermyon Cloud, check out our [develop a Spin application](./develop.md) article to learn how to build a Spin application and then our [deploy a Spin application article](./deploy.md) for deployment instructions. 
+
+## Select Your Spin Application
+
+In the [Fermyon Cloud Dashboard](https://cloud.fermyon.com), navigate to your Spin application of choice and click the "Manage Domains" button.
+
+![DRAFT SCREENSHOT - app panel view](/static/image/custom-domains/app-panel-view.png)
+
+## Input Your Custom Domain
+
+Select the "Add custom domain" button and input your desired domain name. 
+
+![DRAFT SCREENSHOT - step 1](/static/image/custom-domains/Step-1.png)
+
+
+Please add the complete domain name (i.e., `www.example.com`) as Fermyon Cloud does not support redirects at this time. Click save to confirm you would like to add this domain name to Fermyon Cloud.
+
+![DRAFT SCREENSHOT - step 2](/static/image/custom-domains/Step-2.png)
+
+Up next, you'll need to configure your DNS records with your registrar to point at Fermyon's name servers (NS). 
+
+>>If you click "I'll do this later" you will be pausing the workflow and your *custom domain will not work yet* with your Spin application.
+
+![DRAFT SCREENSHOT - step 3](/static/image/custom-domains/Step-3.png)
+
+## Update Your Domain Registrar With Fermyon Cloud Name Servers
+
+If you've never delegated this domain to Fermyon Cloud before, you'll proceed strate to updating your nameserver records with your domain registrar with the nameservers generated by Fermyon Cloud. Each domain registrar has it's own unqiue set of instructions for completing this workflow. Please visit your domain registrar's documentation for instructions. 
+
+![DRAFT SCREENSHOT - step 4](/static/image/custom-domains/Step-4.png)
+
+In addition to generating NS records, Fermyon Cloud will generate a Let's Encrypt certificate on your behalf. 
+
+### Repeat Delegation Only: Update Domain Registrar with TXT Record
+If you've delegate the same domain before with Fermyon Cloud, you will be prompted to add a TXT record as well as part of the verification process. 
```

Comment:
As well as what - is this hinting that the NS step applies here as well?

Reply:
Yes it is, but we are removing TXT validation from MVP so this section is no longer valid